tensorlayer/utils.py

```diff
@@ -542,8 +553,7 @@ def exit_tensorflow(sess=None, port=6006):
         logging.info('OS X: %s' % text)
         subprocess.Popen("lsof -i tcp:" + str(port) + "  | grep -v PID | awk '{print $2}' | xargs kill", shell=True)  # kill tensorboard
     elif _platform == "win32":
-        logging.info(text2 + "Windows")
-        # TODO
+        raise NotImplementedError()
```

Comment:
better to add a string telling "this function is not supported on the Windows platform"